smallfile-wrapper/smallfile-wrapper.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import elasticsearch
+import os
+import sys
+import json
+
+
+# The purpose of this function is to consolidate all result jsons in to 1 list
+def _load_result_JSON(smallfile_path, operations):
+    results = []
+    for operation in operations:
+        file_path = smallfile_path + "_" + operation
+        file_path = "sample_smallfile_OP_from_a_Container"
```

Comment:
looks like you're overwriting `file_path` variable here, probably not intended?

Reply:
It was not at all intended, just added for testing purposes. Have to remove it.